README.md

```diff
@@ -78,21 +78,42 @@ This project uses [CMake] and [Hunter] package manager.
    mkdir build; cd build
    ```
 
-2. Configure the project with CMake. Check out additional
+1. Configure the project with CMake. Check out additional
    [configuration options](#cmake-configuration-options).
 
    ```sh
    cmake ..
    ```
+   e.g. incantation for Ubuntu 17.04 "Zesty" without CL, and addressing the
+   problems of requiring an older version of gcc and error "relocation R_X86_64_32S
+   against.bss' can not be used when making a shared object" mentioned in [this
+   issue](https://github.com/ethereum-mining/ethminer/issues/38).
+   ```sh
+   sudo apt install nvidia-opencl-dev g{cc,++}-5 && \
+   cmake .. \
+    -DETHASHCL=OFF \
+    -DCUDA_NVCC_FLAGS="-ccbin gcc-5" \
+    -DCMAKE_CXX_FLAGS="-fPIC"
+   ```
+
+   Alternatively, the same as above with CUDA enabled:
+   ```sh
+   sudo apt install nvidia-{cuda-toolkit,opencl-dev} g{cc,++}-5 && \
+   cmake .. \
+    -DETHASHCUDA=ON \
+    -DETHASHCL=OFF \
+    -DCUDA_NVCC_FLAGS="-ccbin gcc-5" \
+    -DCMAKE_CXX_FLAGS="-fPIC"
+   ```
 
-3. Build the project using [CMake Build Tool Mode]. This is a portable variant
+1. Build the project using [CMake Build Tool Mode]. This is a portable variant
```

Comment:
Why have you changed these numbers?

Reply:
Github's markdown generator expects any integer and ignores the value when composing ordered lists, so it's not necessary (and probably more laborious) to manually number the markdown itself.

Reply:
Not everybody reads it through GitHub. Bring it back.

Reply:
Hello?